Doc/library/compression.zstd.rst

```diff
@@ -247,6 +247,25 @@ Compressing and decompressing data in memory
       The *mode* argument is a :class:`ZstdCompressor` attribute, either
       :attr:`~.FLUSH_BLOCK`, or :attr:`~.FLUSH_FRAME`.
 
+   .. method:: set_pledged_input_size(size)
+
+      Specify the amount of uncompressed data *size* that will be provided for
+      the next frame. *size* will be written into the frame header of the next
+      frame unless :attr:`CompressionParameter.content_size_flag` is ``False``
+      or ``0``. A size of ``0`` means that the frame is empty. If *size* is
+      ``None``, the frame header will record the size as unknown.
+
+      If :attr:`~.last_mode` is not :attr:`~.FLUSH_FRAME`, a
```

Comment:
What is the difference between `` :attr:`~.last_mode` ``, `` :attr:`.last_mode` `` and simply `` :attr:`last_mode` ``?

Reply:
Good question! I know `~` means "only show the last segment". I believe `` :attr:`last_mode` `` can conflict with a global namespace for names (e.g. it's bad to leave things un-prefixed for `compress` as that could mean the module-level `compress` or the method `ZstdCompressor.compress`).

I'm sure @AA-Turner would have a better explanation than I would though.